src/sentry/users/models/user_avatar.py

```diff
@@ -63,7 +63,7 @@ def outboxes_for_update(self, shard_identifier: int | None = None) -> list[Contr
         )
 
     @contextlib.contextmanager
-    def _maybe_prepare_outboxes(self, *, outbox_before_super: bool):
+    def _maybe_prepare_outboxes(self, *, outbox_before_super: bool) -> Generator[None]:
```

Comment:
Not rlly familiar with the convention here, should this be `Generator[None, None, None] from following [this](https://docs.python.org/3/library/typing.html#typing.Generator), or does it not matter ?

Reply:
`Generator[None]` is the simpler form and one we should use. Recently all `Generator[none, none, none]` were updated. See 16b75635ac031bb5369673ace4ecdf016961a394